control-plane/pkg/reconciler/broker/broker_test.go

```diff
@@ -2085,6 +2189,22 @@ func useTable(t *testing.T, table TableTest, env *config.Env) {
 			expectedTopicDetail = td.(sarama.TopicDetail)
 		}
 
+		externalTopicUsage := false
+		expectedTopicName := fmt.Sprintf("%s%s-%s", TopicPrefix, BrokerNamespace, BrokerName)
+		if _, ok := row.OtherTestData[externalTopic]; ok {
+			expectedTopicName = ExternalTopicName
+			externalTopicUsage = true
+		}
```

Comment:
I would use the value associated with `OtherTestData[externalTopic]` to retrieve the name instead of using the global constant here.

Reply:
```suggestion
		expectedTopicName := fmt.Sprintf("%s%s-%s", TopicPrefix, BrokerNamespace, BrokerName)
		if t, ok := row.OtherTestData[externalTopic]; ok {
			expectedTopicName = t
			externalTopicUsage = true
		}
```

Reply:
ok, true used now `expectedTopicName = t.(string)`